app/models/claim/matching_attribute_finder.rb

```diff
@@ -1,29 +1,52 @@
 class Claim
   class MatchingAttributeFinder
-    ATTRIBUTE_GROUPS_TO_MATCH = [
-      ["teacher_reference_number"],
+    STUDENT_LOANS_AND_ADDITIONAL_PAYMENTS_GROUP = [
```

Comment:
I think this configuration location could be missed and may be best configured in the Policies namespaces.